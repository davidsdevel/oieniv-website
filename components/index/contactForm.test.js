import React from "react";
import ContactForm from "./contactForm";
import renderer from "react-test-renderer";

/*eslint-disable-next-line*/
test("Render ContactForm without crash", () => {
	const component = renderer.create(
		<ContactForm/>,
	);
	let tree = component.toJSON();

	expect(tree).toMatchSnapshot(); // eslint-disable-line
});

