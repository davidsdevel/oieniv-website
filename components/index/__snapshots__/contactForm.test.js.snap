// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render ContactForm without crash 1`] = `
<div>
  <div
    id="contact-container"
  >
    <h3
      className="sub-titles"
    >
      Contactanos
    </h3>
    <input
      name="name"
      onChange={[Function]}
      placeholder="Nombre"
      type="text"
    />
    <input
      name="email"
      onChange={[Function]}
      placeholder="Correo"
      type="email"
    />
    <input
      name="phoneNumber"
      onChange={[Function]}
      placeholder="Numero de Telefono"
      type="number"
    />
    <textarea
      name="comment"
      onChange={[Function]}
      placeholder="Comentario"
    />
    <button
      onClick={[Function]}
    >
      Enviar
    </button>
  </div>
  <style
    jsx="true"
  >
    
					#contact-container {
						width: 100%;
					}
					#contact-container input,
					#contact-container textarea,
					#contact-container button
					{
						display: block;
						margin: 20px auto;
						width: 30%;
						height: 30px;
					}
				
  </style>
</div>
`;
